client/tx/root.go

```diff
@@ -2,12 +2,19 @@ package tx
 
 import (
 	"github.com/spf13/cobra"
+	wire "github.com/tendermint/go-wire"
 )
 
+// type used to pass around the provided cdc
+type commander struct {
```

Comment:
hmm this `commander` paradigm feels a bit weird. I guess the alternative is to have functions like `searchTxCmdFn` that takes a `cdc` and returns a cobra run func. Hm. Guess it's fine for now

Reply:
commander begins to make more sense when you pass more than just the cdc